src/Tokenizer/TokensAnalyzer.php

```diff
@@ -461,6 +461,7 @@ public function isBinaryOperator($index)
                 T_SR => true,                   // >>
                 T_SR_EQUAL => true,             // >>=
                 T_XOR_EQUAL => true,            // ^=
+                CT::T_TYPE_ALTERNATION => true, // |
```

Comment:
Should this be added to the array conditionally? Because it's a PHP 7.1 only feature

Reply:
should be good here, thank for .7 target PR btw :)